frontend/src/components/SQRForm/Fields.vue

```diff
@@ -0,0 +1,171 @@
+            <v-expansion-panel-content>
+              For every issue you wish to report for the SE’s attention, please fill this part as follows.
+
+              <ul>
+                <li>
+                  Enter the Beginning and Ending timing of the section in (h:)mm:ss format.
+                  - Choose the specific issue from the options listed or enter a different issue by selecting ‘Other’.
+                </li>
+                <li>
+                  Please describe the issue in the ‘Description’ field.
+                </li>
+                <li>
+                  For instance, from 20:20 - 21:34 if there is loud noise of roadside vehicles, making it difficult to hear
+                  what Srila Gurudeva is speaking, then please write ‘20:20’ in the Beginning field and ‘21:34’ in the Ending
+                  field. Choose the option ‘Background noise’ in Type and in ‘Description’ field, write ‘Sound of vehicles
+                  honking and general traffic noise.’
+                </li>
+                <li>
+                  Add block by clicking on the green button ‘+ SOUND ISSUE’.
+                </li>
+                <li>
+                  Delete a block by clicking the red 'Bin' icon on the top right of each block.
+                </li>
+              </ul>
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </v-col>
+
+      <v-col class="my-2" cols="12">
+        <h3>E. Total Duration of the Recording</h3>
+        <duration :form="form" :updateForm="updateForm" :removeField="removeField"></duration>
+        <v-expansion-panels class="pt-3">
+          <v-expansion-panel>
+            <v-expansion-panel-header class="pl-0">Guidelines</v-expansion-panel-header>
+            <v-expansion-panel-content>
+              Here, we simply want to know how much the tape has relevant recording.
+              In other words, whether any part of the sound file is blank or inaudible and hence to be discarded.
+              Usually such parts are present towards the end of the file. There might be small parts 5-7 min long
+              in between two lecture recordings, but these can be ignored. Please write the beginning and ending timings
+              of the overall recording in this field in (h:)mm:ss format.
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </v-col>
+
+      <v-col class="my-2" cols="12">
+        <h3>F. Comments</h3>
+        <text-area
+          :form="form"
+          :updateForm="updateForm"
+          :removeField="removeField"
+          pathOverride="comments"
+          :fieldProps="{
+            filled: true,
+            outlined: true,
+            required: true
+          }"
+        ></text-area>
+        <v-expansion-panels class="pt-3">
+          <v-expansion-panel>
+            <v-expansion-panel-header class="pl-0">Guidelines</v-expansion-panel-header>
```

Comment:
I have an idea. Why not to extract `Guidelines` component with a slot for content?